docs/pages/includes/config/oss-enterprise.mdx

```diff
@@ -0,0 +1,750 @@
+## teleport.yaml
```

Comment:
Is this the text of the config reference as it was before this change, or should I review this text along with the rest of the PR?

Reply:
I lot has changed since you first asked this. I ran through a diff looking for changes, and only found changes that have occurred upstream since the PR was staged. I'll update this version to match.